src/features/auth/login/Login.tsx

```diff
@@ -41,21 +42,26 @@ export const Login: FunctionComponent = function () {
   const [errorMessage, setErrorMessage] = useSafeState<string | null>(null)
   const signIn = useSignIn()
   const { isLoggedIn } = useAuthContext()
-
+  const { data } = useUserProfileInfo({
+    cacheTime: 0,
+  })
```

Comment:
Normalement Prettier accepte que cela soit sur une ligne : 
```suggestion
  const { data } = useUserProfileInfo({ cacheTime: 0 })
```